packages/grafana-data/src/utils/url.ts

```diff
@@ -48,7 +48,8 @@ function toUrlParams(a: any) {
     if (typeof v !== 'boolean') {
       s[s.length] = encodeURIComponentAsAngularJS(k, true) + '=' + encodeURIComponentAsAngularJS(v, true);
     } else {
-      s[s.length] = encodeURIComponentAsAngularJS(k, true);
+      s[s.length] =
+        encodeURIComponentAsAngularJS(k, true) + (v ? '' : '=' + encodeURIComponentAsAngularJS('false', true));
```

Comment:
nitpick: I still think this code can be made more readable, maybe extract the value on a separate line?

Reply:
aight, done

Reply:
Love it ❤️ , thank you for contributing to Grafana